can/interfaces/ics_neovi/neovi_bus.py

```diff
@@ -68,6 +69,9 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 open_lock = FileLock(os.path.join(tempfile.gettempdir(), "neovi.lock"))
 description_id = cycle(range(1, 0x8000))
 
+ICS_EPOCH = time.strptime("1/1/2007", "%m/%d/%Y")
+ICS_EPOCH_DELTA = time.mktime(ICS_EPOCH) - time.mktime(time.gmtime(0))
```

Comment:
`time.mktime(time.gmtime(0))` causes an OverflowError for me.

Maybe it's better to use datetime to calculate the delta:
```python3
ICS_EPOCH_DELTA = (
    datetime.fromisoformat("2007-01-01") - datetime.fromisoformat("1970-01-01")
).total_seconds()
```